SD2019StackOverflowApp/src/main/java/ro/utcn/sd/mca/SD2019StackOverflowApp/exception/InvalidSOUserIdException.java

```diff
@@ -0,0 +1,8 @@
+package ro.utcn.sd.mca.SD2019StackOverflowApp.exception;
+
+public class InvalidSOUserIdException extends Exception {
```

Comment:
In general, I would recommend using RuntimeExceptions to now clutter all your code with `throws` declarations :)